backend/coins/btc/account.go

```diff
@@ -603,6 +610,17 @@ func (account *Account) Balance() (*accounts.Balance, error) {
 	return account.transactions.Balance(), nil
 }
 
+func (account *Account) incAndEmitSyncCounter() {
+	if !account.synced {
```

Comment:
It seems like `synced` is updated in multiple places, some of which look like the they run in separate goroutines. If so, wouldn't it need to be guarded by a lock?

I see `Account` already embeds `locker.Locker` but it's not clear at all what it guards. Grepping its usage shows `Initialize`, `updateFeeTargets` methods and some closure funcs here and there. Can't tell for sure.

Reply:
If `synced` really needs a lock, there are a few other places to update, so maybe in a separate commit/PR, but I wish the mutex embeded in the `Account` had some docs.

Reply:
Made an issue for it as it is a bit bigger than this PR.

Reply:
:+1: